shared/tasks/files.mjs

```diff
@@ -11,25 +11,25 @@ import slash from 'slash'
  * Delete path globs for a given destination
  *
  * @param {string} pattern - Pattern to remove
- * @param {AssetEntry[1]} options - Asset options
+ * @param {Pick<AssetEntry[1], "destPath">} options - Asset options
```

Comment:
The clean task only needs a `destPath` so we're being quite strict here

Other properties can be passed in when not TypeScript `"strict": true` but they'll be ignored